packages/govtnz-ds-website/src/commons/styles/elements-global.scss

```diff
@@ -109,6 +109,15 @@ body {
     display: inline;
   }
 
+  // so JS users can't see the after links
+  .no-after-link-js {
+    body {
```

Comment:
Is the `body` part of this irrelevant?

Reply:
No I don't  think so I was going off the code underneath which uses body but it doesn't need it. Have removed it